scripts/cspell/project-words.txt

```diff
@@ -6,7 +6,6 @@ ashmita
 autoscale
 autoscaler
 bansal
-behaviour
```

Comment:
we already removed this in merged PR https://github.com/jaegertracing/documentation/pull/706/files#diff-8cef5df19852ead5efdbe7bbcc1a6293c15d06b34208499964a9ad1da3c4e5b1

please make sure you have up to date main branch